src/test/java/org/betonquest/betonquest/modules/logger/custom/ChatLogFormatterTest.java

```diff
@@ -0,0 +1,76 @@
+package org.betonquest.betonquest.modules.logger.custom;
+
+import org.betonquest.betonquest.api.config.QuestPackage;
+import org.betonquest.betonquest.modules.logger.BetonQuestLogRecord;
+import org.betonquest.betonquest.modules.logger.util.BetonQuestLoggerService;
+import org.bukkit.plugin.Plugin;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.logging.Level;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * A test for the {@link ChatLogFormatter}.
+ */
+@ExtendWith(BetonQuestLoggerService.class)
+class ChatLogFormatterTest {
+    /**
+     * The mocked plugin instance.
+     */
+    private final Plugin plugin;
+
+    /**
+     * Default constructor.
+     */
+    public ChatLogFormatterTest() {
+        plugin = mock(Plugin.class);
```

Comment:
The Plugin mock should be reset() after every test.

```java
@BeforeEach
void prepareMocks() {
    when(plugin.getName()).thenReturn("BetonQuest");
}

@AfterEach
void resetMocks() {
    reset(server);
}
```

An alternative would be to use the `@Mock` annotation:
https://javadoc.io/static/org.mockito/mockito-core/4.3.1/org/mockito/Mockito.html#mock_annotation